unit-tests/src/test/scala/java/util/FormatterSuite.scala

```diff
@@ -0,0 +1,4001 @@
+package java.util
+
+// Ported from Harmony
+
+// Tests related to SecurityManager are removed because they doesn't exist on Scala Native
```

Comment:
We're not going to support security management, so this comment should be dropped.